### PR TITLE
Fix for datatables bug

### LINKED ISSUE #935: Table sorting and filtering is broken for facets that have a taxonomy
Maybe related to issue #873 , and probably because there is an extra column for facets that have a taxonomy like categories and labels.

https://ru.openfoodfacts.org/packaging --> works
https://ru.openfoodfacts.org/labels --> does not work, JS Error:

Uncaught TypeError: Cannot read property 'mData' of undefined
    at HTMLTableCellElement.<anonymous> (datatables.min.js:102)

### PR DESCRIPTION
**Pull request by hangy**
This should fix #935. I don't have a running test instance of OFF right now, but the Perl syntax seems to be OK. Could you test this, please, @stephanegigandet? 😃

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -1560,7 +1565,7 @@ oTable = \$('#tagstable').DataTable({
 	order: [[ 1, "desc" ]],
 	columns: [
 		null,
-		{ "searchable": false }
+		{ "searchable": false } $extra_column_searchable
```
Comment (nicolasleger) [inline lib/ProductOpener/Display.pm]: You might have forgotten to remove the content now inside the variable.

Comment (stephanegigandet) [reply lib/ProductOpener/Display.pm]: looks good to me

Comment (hangy) [reply lib/ProductOpener/Display.pm]: Thanks!

@nicolasleger The idea was that `$extra_column_searchable` is either empty or contains the entry for the third column. Not the cleanest way to implement this - I'll admit 😄 - but it should work. 😉

Comment (nicolasleger) [reply lib/ProductOpener/Display.pm]: Ok no problem so ! I was in doubt, sorry. Good 👍🙂

Comment (hangy) [reply lib/ProductOpener/Display.pm]: No problem. Thanks for checking! :)

**Pull request merged**